tests/integration/test_culling.py

```diff
@@ -1,17 +1,17 @@
 import time
 
 import pytest
+from kubernetes.dynamic import Resource
 
 from tests.integration.utils import find_resource
 
 
 @pytest.mark.culling
```

Comment:
pytest warns that this mark is unknown.

AFAICT, it's defined nowhere in this branch nor default.

Reply:
Ok cool I will add it in pytest. These tests take a bit longer to run so it is good to be able to run them separately if needed.